metricbeat-tests/runner_test.go

```diff
@@ -248,15 +260,15 @@ func retrySearch(stackName string, indexName string, esQuery map[string]interfac
 			return result, nil
 		}
 
-		log.WithFields(log.Fields{
-			"attempt":       attempts,
-			"errorCause":    err.Error(),
-			"index":         indexName,
-			"query":         esQuery,
-			"retryAttempts": queryMaxAttempts,
-			"retryTimeout":  queryRetryTimeout,
-		}).Warnf("Waiting %d seconds for the index to be ready", queryRetryTimeout)
 		if attempts > 1 {
+			log.WithFields(log.Fields{
+				"attempt":       attempts,
+				"errorCause":    err.Error(),
+				"index":         indexName,
+				"query":         esQuery,
+				"retryAttempts": queryMaxAttempts,
+				"retryTimeout":  queryRetryTimeout,
+			}).Warnf("Waiting %d seconds for the index to be ready", queryRetryTimeout)
```

Comment:
@mdelapenya wdyt about changing this to debug too, this is something that is always going to happen, I don't think it worths to be logged at the warn level.

Reply:
Agreed

Reply:
Done